_config.yml

```diff
@@ -150,6 +150,17 @@ fetch-remote:
           - "!docs/README.md" # readme to make things nice in the compose-cli repo, but meaningless here
           - "!docs/architecture.md" # Compose-CLI architecture, unrelated to cloud integration
 
+  - repo: "https://github.com/docker/buildx"
+    ref: "master"
+    paths:
+      - dest: "build/guides"
+        src:
+          - "docs/guides/*.md"
+          - "!docs/guides/cicd.md" # CI/CD documentation is already handled in ci-cd/github-actions
```

Comment:
I'm not sure what we're doing here, is this about removing something from the docs that is some repo? If so, I'm not sure that is the right way forward.

Reply:
We are fetching markdown files from buildx repo in https://github.com/docker/buildx/tree/master/docs/guides (except ci-cd) and add them to `build/guides` folder here.
Will be available in `Manuals > Docker Build > Guides`.